src/Components/OggettiComponents/Oggetto.js

```diff
@@ -1,4 +1,4 @@
-import ChangeQuantity from "../Cart/ChangeQuantity";
+import ChangeQuantity from "./ChangeQuantity";
```

Comment:
I moved the file to the `OggetiComponents` folder since the "Change Quantity" is used under each object, so it seems like it fits better within this group rather than within the Cart directory.